llvm/lib/Linker/IRMover.cpp

```diff
@@ -1757,6 +1784,22 @@ bool IRMover::IdentifiedStructTypeSet::hasType(StructType *Ty) {
   return I == NonOpaqueStructTypes.end() ? false : *I == Ty;
 }
 
+void IRMover::LibcallHandler::updateLibcalls(const llvm::Triple &Triple) {
```

Comment:
```suggestion
void IRMover::LibcallHandler::updateLibcalls(const Triple &Triple) {
```

Reply:
Doesn't work because the variable is named `Triple`. I could rename it to `TheTriple` if you'd prefer.